config.go

```diff
@@ -42,6 +44,8 @@ var defaultConfig = Config{
 	TunaMeasureBandwidth:   false,
 	TunaMeasureStoragePath: "",
 	SessionConfig:          nil,
+	ReconnectRetries:       -1,
```

Comment:
Default value has to be 0, because 0 cannot overwrite non 0 value

Reply:
Yes, modified